crates/bin/src/args.rs

```diff
@@ -91,22 +101,7 @@ pub struct Args {
     /// - zip: Download format is Zip
     ///
     /// - bin: Download format is raw / binary
-    #[clap(
-        help_heading = "Overrides",
-        long,
-        value_name = "PKG_FMT",
-        possible_values = [
-            PossibleValue::new("tar").help(
-                "Download format is TAR (uncompressed)."
-            ),
-            PossibleValue::new("tbz2").help("Download format is TAR + Bzip2."),
-            PossibleValue::new("tgz").help("Download format is TGZ (TAR + GZip)."),
-            PossibleValue::new("txz").help("Download format is TAR + XZ."),
-            PossibleValue::new("tzstd").help("Download format is TAR + Zstd."),
-            PossibleValue::new("zip").help("Download format is Zip."),
-            PossibleValue::new("bin").help("Download format is raw / binary."),
-        ]
-    )]
+    #[clap(help_heading = "Overrides", long, value_name = "PKG_FMT")]
```

Comment:
I can't comment on the line but there's a typo in the comment above: availiable -> available.

Also might be worth trying out if they've fixed the list formatting without putting newlines between each item

Reply:
> I can't comment on the line but there's a typo in the comment above: availiable -> available.

Fixed

> Also might be worth trying out if they've fixed the list formatting without putting newlines between each item

Unfortunately, no.